schema_editor/app/scripts/resources/boundaries-service.js

```diff
@@ -0,0 +1,29 @@
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function Boundaries($resource, Config) {
+        var urlString = Config.api.hostname + '/api/boundaries/:id';
+
+        var resource = {
+          'query': stdResource.query,
+          'get': stdResource.get,
+          'post': postWithFile,
+          'remove': stdResource.remove,
+          'delete': stdResource.delete
+        };
+
+        var stdResource = $resource(urlString, {id: '@uuid'}, {
+            query: {
+                method: 'GET',
+                transformResponse: function(data) { return angular.fromJson(data).results; },
```

Comment:
Since this is doing something non-standard, you'll want to add a test to ensure query returns what is expected. I can point you to a few service/http test examples if you can't find one.